libkbfs/folder_block_ops.go

```diff
@@ -2498,8 +2518,8 @@ func (fbo *folderBlockOps) searchForNodesInDirLocked(ctx context.Context,
 		if _, ok := nodeMap[de.BlockPointer]; ok {
 			childPath := currDir.ChildPath(name, de.BlockPointer)
 			// make a node for every pathnode
-			var n Node
-			for _, pn := range childPath.path {
+			n := rootNode
+			for _, pn := range childPath.path[1:] {
```

Comment:
Yes, exactly.  I figured since we were passing it around now, might as well use it.